documentation/slate/source/includes/apis.md

```diff
@@ -1936,8 +1937,12 @@ For example:
 
 -   overall public transport traffic report on Ile de France coverage
     -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports>
+-   overall public transport traffic report on Ile de France coverage with disruptions having tags passed in parameter values
+    -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports?tags[]=bob&tags[]=bobette>
 -   Is there any perturbations on the RER network ?
     -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports>
+-   Is there any perturbations on the RER network with disruptions having tags passed in parameter values ?
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports?tags[]=bob&tags[]=bobette>
```

Comment:
Same comment for Bob and Bobette. Even though they're nice and welcome in our tests, it doesn't fell like they're in the right place here.